problem-gson/src/test/java/org/zalando/problem/ProblemAdapterFactoryTest.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.problem;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ProblemAdapterFactoryTest {
+
+    @Test
+    void defaultConstructorShouldBuildIndexCorrectly() {
+        new ProblemAdapterFactory();
+    }
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem/pullRequest?prid=4476243)